test/utils.js

```diff
@@ -0,0 +1,70 @@
+import { watchFile, readdirSync, statSync } from 'fs';
```

Comment:
I didn't do as good a job as I should have when taking Mocha's code and converting styles